<launch>

<include file="$(find realsense2_camera)/launch/rs_camera.launch"/>

<node pkg="stalk_detect" name="StalkDetect" type="stalk_detect" output="screen"/>

</launch>